content/java/concepts/calendar/terms/clone/clone.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.clone()'
+Description: 'Returns a copy of a Calendar object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.clone()`** method of the `Calendar` class is used to return a copy of a Calendar object.
+
+## Syntax
+
+```pseudo
+calendar.clone()
+```
+
+Returns the copy of the calendar object.
+
+## Example
+
```

Comment:
The example should have an introductory sentence summarizing what the example does.